### PR TITLE
add anglePopUp

### PR DESCRIPTION
**Pull request by MoElnaggar14**

### CODE REVIEW

**Comment by jjochen** [comment]
Thank you for the PR! I'll check it out the moment I find some time.

File: Sources/AnimationConfiguration.swift [hunk 1]
```diff
@@ -465,6 +510,18 @@ internal extension JJItemAnimationConfiguration {
             return startAngle + CGFloat(index) * (endAngle - startAngle) / (CGFloat(numberOfItems) - 1)
         }
     }
+    
+    static func angleForItemForAngle(at index: Int, numberOfItems: Int, actionButton: JJFloatingActionButton) -> CGFloat {
```
Comment (jjochen) [inline Sources/AnimationConfiguration.swift]: `actionButton` is not needed, right?

File: Sources/AnimationConfiguration.swift [hunk 2]
```diff
@@ -327,6 +347,31 @@ import Foundation
...
         }
     }
+    
+    /// Returns an item layout object that places the items in angle with given radius.
+    ///
+    /// - Parameter radius: The distance between the center of an item and the center of the button itself.
+    ///
+    /// - Returns: An item layout object.
+    ///
+    @objc static func angle(center: CGPoint, radius: CGFloat = 100) -> JJItemLayout {
```
Comment (jjochen) [inline Sources/AnimationConfiguration.swift]: `center` is not used, correct?

File: Sources/AnimationConfiguration.swift [hunk 3]
```diff
@@ -251,6 +252,25 @@ import Foundation
...
+    ///   - `itemLayout = .circular()`
+    ///   - `closedState = .scale()`
+    ///
+    /// - Parameter center: The position point of the button itself.
+    ///
+    /// - Parameter radius: The distance between the center of an item and the center of the button itself.
+    ///
+    /// - Returns: An item animation configuration object.
+    ///
+    @objc static func anglePopup(center: CGPoint, radius: CGFloat = 100) -> JJItemAnimationConfiguration {
```
Comment (jjochen) [inline Sources/AnimationConfiguration.swift]: I don't think `center` is needed.

**Comment by jjochen** [comment]
Hi Mohammed, currently if I add 
```
        actionButton.itemAnimationConfiguration = .anglePopup()
``` 
to the [BasicsExampleViewController](https://github.com/jjochen/JJFloatingActionButton/blob/e0dfa2a6bb762b1e88d68d9489b7864e0b91be5c/Example/JJFloatingActionButton/BasicsExampleViewController.swift#L33) I get this:

Simulator Screen Shot

Is this what you were aiming for? I think it's a quite a special case and I wouldn't add it directly to the library. It would probably make more sense to add it to you project as an extension of `JJItemAnimationConfiguration`.

**Comment by MoElnaggar14** [comment]
it should start with Button's centre point, so I will fix it and push the right changes 
`.anglePopup()` should take a CGPoint Parameter of button

**Pull request closed without merging**